### PR TITLE
SCAL-234286 added flip tooltip context menu sdk flag

### PR DESCRIPTION
**Pull request by ubaid-ahmed-ts**

### CODE REVIEW

File: src/embed/app.ts [hunk 1]
```diff
@@ -459,6 +459,13 @@ export interface AppViewConfig extends Omit<ViewConfig, 'visibleTabs'> {
      * ```
      */
     isUnifiedSearchExperienceEnabled?: boolean;
+    /**
+     * Flag to control new flip tooltip context menu experience
+     *
+     * @default false
+     * @version SDK: 1.35.0 | Thoughtspot: 10.6.0.cl
```
Comment (adityamittal3107) [inline src/embed/app.ts]: 10.6.0.cl should point to 1.36

File: src/embed/app.ts [hunk 2]
```diff
@@ -579,6 +587,7 @@ export class AppEmbed extends V1Embed {
         params[Param.ModularHomeExperienceEnabled] = modularHomeExperience;
         params[Param.CollapseSearchBarInitially] = collapseSearchBarInitially || collapseSearchBar;
         params[Param.EnableCustomColumnGroups] = enableCustomColumnGroups;
+        params[Param.EnableFlipTooltipToContextMenu] = enableFlipTooltipToContextMenu;
```
Comment (adityamittal3107) [inline src/embed/app.ts]: Why are you adding this flag by default in iframe URL?
Also, is this flag specific to appEmbed only?

File: src/embed/app.ts (hunk 1 above)
Comment (ubaid-ahmed-ts) [reply src/embed/app.ts]: okay

File: src/embed/app.ts (hunk 2 above)
Comment (ubaid-ahmed-ts) [reply src/embed/app.ts]: i will remove default addition

And yes it is needed in other embed also, will add it there also thanks

File: src/embed/app.spec.ts [hunk 3]
```diff
@@ -479,6 +479,36 @@ describe('App embed tests', () => {
         });
     });
 
+    test('Should add flipTooltipToContextMenuEnabled flag to the iframe src', async () => {
```
Comment (adityamittal3107) [inline src/embed/app.spec.ts]: If it exists in all the classes write in a common point, instead of writing it seperately for all classes.

Comment (ubaid-ahmed-ts) [reply src/embed/app.spec.ts]: can you please point me out on which file is common for these?

Comment (ubaid-ahmed-ts) [reply src/embed/app.spec.ts]: Done

**Review by adityamittal3107 — APPROVED** [review]

**Pull request merged**